resources/zh/cemu.po

```diff
@@ -452,76 +555,36 @@ msgstr "设置CPU模拟模式"
 
 #: GameProfileWindow.cpp
 msgid "Thread quantum"
-msgstr "Thread quantum"
+msgstr "线程量子长度为"
```

Comment:
This gets translated as "The thread quantum length is" by GTranslate/DeepL, is that true?
If it is, I think it's better to keep the style of the original string (so something like "thread quantum length")

Reply:
Yes. Because in Cemu, the original text here is "Thread quantum 20000 cycles", in order to make it more fluent in Chinese, I added is to make it conform to Chinese reading habits. If you think this is inappropriate it can be corrected, but will not read fluent.